src/ceph_fuse.cc

```diff
@@ -295,6 +295,8 @@ int main(int argc, const char **argv, const char *envp[]) {
     //cout << "child done" << std::endl;
     return r;
   } else {
+    if (restart_log)
```

Comment:
If you want to try and put together something better that'd be welcome. But I think we need a fix pretty much now and I'm not familiar enough with the init code to make those kinds of changes, @tchaikov.